Ubuntu/installRavKav.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+sudo apt-get install pcscd wine
+sudo cp ravkav "/usr/bin/ravkav"
```

Comment:
Hi @uda, yes, but it is **managed** by a package. Maybe we can modify this to use `checkinstall`, but to `cp` to `/usr/bin/` is something I really hate. We can also use `/usr/local/bin` if you prefer.